src/main/java/stobberi/command/Command.java

```diff
@@ -0,0 +1,45 @@
+package stobberi.command;
+
+import stobberi.stobberiException.StobberiException;
+
+/**
+ * Represents a command that can be executed. The command has an exit flag that indicates
+ * whether the command should terminate the application or not.
+ */
+public class Command {
```

Comment:
I like how you created Command as a parent class and created subclasses for different commands that all have the execute function. This makes the code much cleaner and there will not be a long switch statement as you are using polymorphism to deal with the different commands. Nice job! Maybe you could declare this class as an Abstract class?